pitch.md

```diff
@@ -23,38 +37,125 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+Like LaTeX the programmer would write text to a file. 
+The programmer types out what would be on each line of the tableau tree,
+using special characters for logic symbols.
+The programmer should also have a way to draw a check mark at the end of each line of the tree.
+Tableau trees are binary trees so the programmer can also specify if the tree branches off.
+
+One possible syntax is
+```
+(A \or B) \and C \check
```

Comment:
Hmmm what do the slashes serve? And since OR is roughly represented by V and AND is roughly represented by ^, would you consider using these characters instead of the words "OR" and "AND"?